src/ServiceControl.Audit/Infrastructure/RavenDB/MigrateKnownEndpoints.cs

```diff
@@ -0,0 +1,88 @@
+﻿using NServiceBus;
+using NServiceBus.Features;
+using Raven.Client;
+using ServiceControl.Audit.Monitoring;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ServiceControl.Audit.Infrastructure.RavenDB
+{
+    class MigrateKnownEndpoints : Feature
```

Comment:
So this works as long as we have the sendonly endpoint still present in the audit instance.

Reply:
Correct. There are other Feature's in the audit instance so I feel it's safe enough as removing the endpoint will be a much larger process and won't likely result in forgotten features running.

I don't like that this is essentially a one-off while all the other features in the audit instance are long-running.

Do you have a suggestion for a better place to put this?

Reply:
Given that we create the endpoint during the upgrade process to run the installers, and that will execute this feature, we could consider moving this to an installer.

Reply:
I'm moving this out of a feature anyway